src/services/repository.ts

```diff
@@ -26,7 +25,7 @@ const kDateFormatter = Intl.DateTimeFormat("en-GB", {
 const kPullUrlPostfixLen = "{/number}".length;
 const kCommitUrlPostfixLen = "{/sha}".length;
 
-export interface DashlogRepository {
+export interface DashlogRepository<PluginsGeneric extends object> {
```

Comment:
I wonder if we should add `any` as a default generic here:
`interface DashlogRepository<PluginsGeneric extends object = any>`
We have multiple `DashlogRepository<any>` that would become `DashlogRepository`
Wdyt ?